openerp/modules/loading.py

```diff
@@ -415,7 +415,10 @@ def load_modules(db, force_demo=False, status=None, update_module=False):
         t0 = time.time()
         t0_sql = openerp.sql_db.sql_counter
         if openerp.tools.config['test_enable']:
-            cr.execute("SELECT name FROM ir_module_module WHERE state='installed'")
+            if update_module:
+                cr.execute("SELECT name FROM ir_module_module WHERE state='installed' and name in %s", (tuple(mods),))
```

Comment:
I've a bug when tests are ran at install, sometimes mods is empty and this changes breaks my tests.. 
```python
if update_module and mods:
    ...
```